ui/components/ui/hardware-wallet-state/hardware-wallet-state.stories.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
```

Comment:
I would consider this an app specific component. It doesn't have a lot of reusability and is for a very specific purpose. It could probably live in `ui/components/app`. There isn't much documentation on `App` vs `UI` components but the difference is that `UI` components should have very high reusability. The folders are currently pretty muddy in terms of definition. Hopefully we can work on improving that.

Reply:
Done - thanks for the explanation. It started in `App`, then I moved it to `UI` and then I had second thoughts :joy: